spec/ParseAPI.spec.js

```diff
@@ -646,6 +646,28 @@ describe('miscellaneous', function () {
       });
   });
 
+  it('pointer reassign on nested fields is working properly', async () => {
```

Comment:
Could you please mark it to no test for postgres?

Reply:
Both done